### PR TITLE
helm install with namespace

### PR DESCRIPTION
**Pull request by baturozgur**

### CODE REVIEW

File: installation/helm.md [hunk 1]
```diff
@@ -19,37 +19,36 @@ Add Edge Delta helm repository
...
-NAMESPACE: default
+LAST DEPLOYED: Fri Jul 17 17:49:42 2020
+NAMESPACE: edgedelta
 STATUS: deployed
 REVISION: 1
 TEST SUITE: None
 NOTES:
 1. Visit https://admin.edgedelta.com
-2. Find the configuration with <API_KEY> to check reporting status
+2. Find the configuration with <API-KEY> to check reporting status
```
Comment (taylanisikdemir) [inline installation/helm.md]: what do we mean exactly by reporting status? It's a little confusing me. At this point they already have API KEY, right? which means they already used admin UI to create a config and copied its id. I don't understand why they would go to admin UI and find the config after deploying with helm. 
Once they deploy with helm, next step would be to visit admin UI and check the active nodes.

**Review by taylanisikdemir — APPROVED** [review]

File: installation/helm.md (hunk 1 above)
Comment (baturozgur) [reply installation/helm.md]: I mean to see 1. visit https://admin.edgedelta.com 2 check agents reporting status for this config.

Comment (baturozgur) [reply installation/helm.md]: If reporting status wording is confusing (agents are reporting properly as up or not there is some/no bytes analyzed) we can just say "check if agents are active". This is helm template output by the way, we need to both update here and Notes.txt in helm chart (in egdedelta/charts repo)

Comment (taylanisikdemir) [reply installation/helm.md]: "check if agents are active" sounds good

**Pull request merged**